clouddriver-web/clouddriver-web.gradle

```diff
@@ -22,6 +22,7 @@ repositories {
 
 dependencies {
   compile project(':clouddriver-aws')
+  compile project(':clouddriver-aws-ecs')
```

Comment:
make this `clouddriver-ecs` to follow the conventions of clouddriver-appengine and clouddriver-gce